lib/CrEOF/Geo/String/Parser.php

```diff
@@ -57,29 +52,48 @@ class Parser
     private $nextSymbol;
 
     /**
+     * @var Lexer
+     */
+    private static $lexer;
```

Comment:
Since the geo-parser will be used as a dependency by other projects, you might end up with two `users` that create their own `Parser()`